localization/ndt_scan_matcher/README.md

```diff
@@ -244,13 +244,17 @@ This is a function that uses no ground LiDAR scan to estimate the scan matching
 Calculate 2D covariance (xx, xy, yx, yy) in real time using the NDT convergence from multiple initial poses.
 The arrangement of multiple initial poses is efficiently limited by the Hessian matrix of the NDT score function.
 In this implementation, the number of initial positions is fixed to simplify the code.
+To obtain the covariance, MULTI_NDT computes until convergence at each initial position, while MULTI_NDT_SCORE uses the nearest voxel transformation likelihood.
```

Comment:
This is confusing because it has not yet been explained what MULTI_NDT and MULTI_NDT_SCORE are here. 
To make it easier for the user to understand, please explain first that they are mode names of covariance estimation.

For example,
```Three modes are provided for 2D covariance estimation: LAPLACE_APPROXIMATION, MULTI_NDT, and MULTI_NDT_SCORE. To obtain the covariance,...```

Reply:
OK! Thanks for pointing that out!